lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -133,22 +138,38 @@ class LAMPApp extends Script {
     }
   }
 
-  // a meta-script that installs all apache/php-related scripts
-  addScriptApachePhp() {
+  // a meta-script that installs all apache/mysql/php related scripts.
```

Comment:
Why is this piggy backing on the apache config? Code looks good here but each function should only do one thing; this should be made into its own function.

Reply:
Fixed.